packages/web-component-library-stencil/tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "jsxFactory": "h",
     "skipLibCheck": true
   },
-  "include": ["../../components/"],
-  "exclude": ["dist", "loader", "node_modules"]
+  "include": ["../../components/", "*/*.ts", "*/*.tsx", "./stencil.config.ts"],
```

Comment:
I suggest to use `**/*.ts` instead of `*/*.ts`, that would include TypeScript files in deeper nested directories too. Might save us some headaches later on.